packages/shared-lib/src/app/types/easy-genomics/laboratory-run.d.ts

```diff
@@ -30,7 +30,7 @@ import { BaseAttributes, RunType } from "../base-entity";
 export interface LaboratoryRun extends BaseAttributes {
   LaboratoryId: string; // DynamoDB Partition Key (String)
   RunId: string; // DynamoDB Sort Key (String) & Global Secondary Index (String)
-  UserId: string; // Global Secondary Index (String)
+  UserId?: string; // Global Secondary Index (String)
```

Comment:
The UserId property is intended to efficiently query the LaboratoryRuns by a User, so this should be a required field.

Reply:
From memory this was causing a Zod validation error due to the `UserId` not being present, but I can't recall in which request. I can't currently replicate so I'll revert this change.